packages/rooks/rollup.future.config.ts

```diff
@@ -0,0 +1,68 @@
+// import esbuild from "rollup-plugin-esbuild";
+// import terser from "rollup-plugin-terser";
+import dts from "rollup-plugin-dts";
+import typescript2 from "rollup-plugin-typescript2";
+import { typescriptPaths } from "rollup-plugin-typescript-paths";
+import pkg from "./package.json";
+
+const external = [
+  ...Object.keys(pkg.dependencies),
+  ...Object.keys(pkg.peerDependencies || {}),
+];
+
+const distDir = "./dist/future";
+
+const tsPathPlugin = typescriptPaths({
+  preserveExtensions: true,
+  tsConfigPath: "./tsconfig.json",
```

Comment:
typescriptPaths plugin does not support extend from tsconfig.build.json. So I use the paths from there. Note that it makes use remove composite: true.